ansible/roles/go-cli/tasks/deploy.yml

```diff
@@ -6,15 +6,39 @@
     path: "{{ go_cli_conf_dir }}"
     state: directory
 
+- name: "Ensure nginx directory for go-cli exists"
+  file:
+    path: "{{ go_cli_nginx_dir  }}"
+    state: directory
+
 - name: "Pull the {{ docker_image_tag }} image of go-cli"
   shell: "docker pull {{ docker_registry }}whisk/go-cli:{{ docker_image_tag }}"
   when: docker_registry != ""
 
-# TODO: Publish compressed binaries using nginx
```

Comment:
- name: "Run Docker"
  shell: "docker run --name go-cli {{ docker_registry }}whisk/go-cli:{{ docker_image_tag }}"

- name: "Copy all the go-cli linux binaries into nginx"
  shell: "docker cp go-cli:/src/github.com/go-whisk-cli/linux/amd64/openwhisk-linux-amd64.tar.gz '{{ nginx_conf_dir }}'"
  ignore_errors: yes